src/actions/compositionModel.ts

```diff
@@ -0,0 +1,19 @@
+import xcomponentapi from "reactivexcomponent.js";
+import { Action } from "redux";
+
+export const INIT_COMPOSITION_MODEL = "INIT_COMPOSITION_MODEL";
+
+export interface GlobalCompositionModelAction extends Action {
+    compositionModel: any;
```

Comment:
The composition model is defined in the api. We have to expose it in the api first